.github/workflows/nightly.yml

```diff
@@ -0,0 +1,76 @@
+name: nightly
+on:
+  pull_request:
+    branches: [master]
+    types: [closed]
+  workflow_dispatch:
+
+jobs:
+  build:
+    strategy:
+      matrix:
+        include:
+          - os: ubuntu-latest
+            target: x86_64-unknown-linux-gnu
+            suffix: ".gz"
+          - os: macos-latest
+            target: x86_64-apple-darwin
+            suffix: ".gz"
+          - os: macos-latest
+            target: aarch64-apple-darwin
+            suffix: ".gz"
+          - os: windows-latest
+            target: x86_64-pc-windows-msvc
+            suffix: ".exe"
+    runs-on: ${{ matrix.os }}
+    if: github.event.pull_request.merged == true)
```

Comment:
Is there a leading `)` on this that isn't needed?

Reply:
Thanks!